### ISSUE TITLE
Percent is not encoded correctly in paths

### ISSUE THREAD

**Issue by tdryer** [gruns/furl#84]
When constructing a path, a reserved character like space is encoded correctly by furl: 

```python
>>> furl('http://localhost').add(path=['foo bar']).url
'http://localhost/foo%20bar'
```

However, a percentage character is not encoded (and the following two characters are capitalized):

```python
>>> furl('http://localhost').add(path=['foo%bar']).url
'http://localhost/foo%BAr'
```

In this case I'd expect the output to be:
```python
'http://localhost/foo%25bar'
```

**Comment by gruns** [comment]
This bug is fixed in the latest furl v0.5.7.

```python
>>> from furl import furl
>>> furl('http://localhost').add(path=['foo bar']).url
'http://localhost/foo%20bar'
>>> furl('http://localhost').add(path=['foo%bar']).url
'http://localhost/foo%25bar'
```

Update with

```
pip install furl --upgrade
```

Thank you for bringing this issue to my attention Tom. Don't hesitate to let me
know if there's anything else I can do for you.

**Comment by tdryer** [comment]
Thanks for the fix!
